icons/behance.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Behance</title><path d="M6.981 4.46c.693 0 1.332.053 1.918.213a4.18 4.18 0 0 1 1.492.586c.426.266.746.639.959 1.119.213.48.32 1.066.32 1.705 0 .746-.16 1.385-.533 1.865-.32.48-.853.906-1.492 1.226.906.266 1.599.746 2.025 1.385a4.2 4.2 0 0 1 .693 2.345c0 .746-.16 1.385-.426 1.918a3.547 3.547 0 0 1-1.172 1.332c-.48.32-1.066.586-1.705.746-.639.16-1.279.266-1.918.266H0V4.46h6.981zm-.427 5.968c.586 0 1.066-.16 1.439-.426.373-.267.533-.746.533-1.332 0-.32-.053-.639-.16-.853s-.266-.373-.48-.533c-.213-.106-.426-.213-.692-.266-.267-.054-.533-.054-.853-.054H3.25v3.464h3.304zm.16 6.288c.32 0 .639-.053.906-.107.266-.053.533-.16.746-.32.213-.16.373-.32.533-.586.107-.266.213-.586.213-.959 0-.746-.213-1.279-.639-1.652-.426-.32-1.012-.48-1.705-.48H3.25v4.103h3.464zm10.285-.053c.426.426 1.066.639 1.918.639.586 0 1.119-.16 1.545-.426.426-.32.693-.639.799-.959h2.611c-.426 1.279-1.066 2.185-1.918 2.771-.853.533-1.865.853-3.091.853-.853 0-1.599-.16-2.291-.426a4.245 4.245 0 0 1-1.705-1.172 4.228 4.228 0 0 1-1.066-1.758c-.266-.693-.373-1.439-.373-2.291 0-.799.107-1.545.373-2.238a5.73 5.73 0 0 1 1.119-1.812c.48-.48 1.066-.906 1.705-1.172a5.988 5.988 0 0 1 2.238-.426c.906 0 1.705.16 2.398.533.693.373 1.226.799 1.652 1.439.426.586.746 1.279.959 2.025.107.746.16 1.492.107 2.345h-7.727c0 .849.32 1.648.747 2.075zm3.357-5.596c-.373-.373-.959-.586-1.652-.586-.48 0-.853.107-1.172.266s-.533.373-.746.586a1.42 1.42 0 0 0-.373.746c-.053.266-.107.48-.107.693h4.796c-.107-.799-.373-1.332-.746-1.705zm-4.69-5.595h5.968v1.439h-5.968V5.472z"/></svg>
```

Comment:
Did you take the SVG visible on the mobile version of their website header?
Not sure if I'm doing something wrong here, but I got a huge difference...
![image](https://github.com/simple-icons/simple-icons/assets/12817388/f68ce21b-c19b-4b44-8257-bbec1c63164b)

Reply:
Wow, what went on THERE?!
I'll have a look at this one either this evening or tomorrow morning.